bioimageio/spec/v0_3/utils.py

```diff
@@ -35,9 +35,9 @@ def _(source: dict, root_path: Optional[pathlib.Path] = None) -> Tuple[raw_nodes
 def _(source: os.PathLike, root_path: Optional[pathlib.Path] = None) -> Tuple[raw_nodes.Model, pathlib.Path]:
     source = pathlib.Path(source)
 
-    suffixes = source.suffixes
-    if len(suffixes) < 2 or suffixes[-1] not in (".yml", ".yaml") or source.suffixes[-2] != ".model":
-        raise ValidationError(f"invalid suffixes {''.join(suffixes)} for source {source}")
+    # suffixes = source.suffixes
+    # if len(suffixes) < 2 or suffixes[-1] not in (".yml", ".yaml") or source.suffixes[-2] != ".model":
```

Comment:
since the addition of the `type` field to the model spec, the ".model" suffix lost it's importance anyway. Let's drop that check and only leave in checking for a valid yaml suffix....

Reply:
So you mean just the following, right?
```python
if suffixes[-1] not in (".yml", ".yaml"):
  raise ValidationError("....")
```